src/openrct2/peep/peep.c

```diff
@@ -9823,10 +9823,8 @@ static sint32 guest_path_find_leaving_park(rct_peep *peep, rct_map_element *map_
 	rct2_peep_spawn* peepSpawn = &gPeepSpawns[0];
 	// Peeps for whatever reason return to their original spawn point
 	// this in future should look for the nearest.
-	if (peep->sprite_index & 1 && gPeepSpawns[1].x != PEEP_SPAWN_UNDEFINED) {
-		peepSpawn++;
-	}
-
+	get_random_peep_spawn(peepSpawn);
```

Comment:
It seems it would, but it hardly seems to even use any other spawn than 0.. if you load dynamite dunes and watch the guests leave the park, they seem to always go straight and never take the forked way. Idk why. It may need multiple tests.. also would we be able to add a new data variable to rct_peep to store the chosen peep spawn, or will it cause problems loading/saving with a different struct size?